scripts/update_all_products.pl

```diff
@@ -1359,21 +1372,24 @@
 
 			# Otherwise, we silently update the .sto file of the last version
 			else {
-
 				# make sure nutrient values are numbers
 				ProductOpener::Products::make_sure_numbers_are_stored_as_numbers($product_ref);
 
+				# Make sure product _id and code are saved as string and not a number
```

Comment:
I really find it a pity that we have this kind of duplicate code instead of Products.pm handling all this (may it with special parameters).